bin/ts-mage

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ts-node
+
+// We only need this file to specify that
+// we should run using ts-node instead of node.
+
+// Require the node version
```

Comment:
I get what you mean now. Maybe change "node" to "JavaScript" or "non-TypeScript"?